### PR TITLE
Fixed bug where moved post links were incorrect

### PR DESCRIPTION
**Pull request by delgiudices**
https://meta.discourse.org/t/when-moving-posts-links-quotes-that-were-moved-too-dont-get-updated/13999/2

### CODE REVIEW

File: app/models/post_mover.rb [hunk 1]
```diff
@@ -91,6 +91,12 @@ def create_first_post(post)
   def move(post)
     @first_post_number_moved ||= post.post_number
 
+    post.replies.each do |reply|
+      reply.raw = reply.raw.sub(/post:[1-9]+/, "post:#{@move_map[post.post_number]}")
```
Comment (SamSaffron) [inline app/models/post_mover.rb]: it should be gsub, but really this is far too wide a catch, what if the number is missing from @move_map? what if it happens not to be a quote and just some other part of the post

**Comment by delgiudices** [comment]
Okay @SamSaffron it nows replace all instances of quotes and only inside [quote ]. About the @move_map, assuming move(post) is called by move_each_post then i think @move_map will be present, how do you think i should handle that case?

**Comment by SamSaffron** [comment]
Sorry, it is still not safe enough, a post can contain quotes to other topics, not only quote in the current topic.

**Comment by coding-horror** [comment]
I think we will have to attack this later after CommonMark support is merged

**Pull request closed without merging**